packages/neuron-ui/src/containers/MainContent/state.ts

```diff
@@ -1,6 +1,13 @@
 import { CapacityUnit } from '../../utils/const'
 
 export const initState = {
+  mnemonic: {
```

Comment:
Does this `initState` mean if our app becomes more complicated and has more sub features, we have to store all local data within `MainContent`?